cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CreateExecutableTask.java

```diff
@@ -127,16 +126,9 @@ public void execute(Project project) {
                 }
             }
 
-            List<Diagnostic> diagnostics = new ArrayList<>(emitResult.diagnostics().diagnostics());
-            if (!diagnostics.isEmpty()) {
-                //  TODO: When deprecating the lifecycle compiler plugin, we can remove this check for duplicates
-                //   in JBallerinaBackend diagnostics and the diagnostics added to EmitResult.
-                diagnostics = diagnostics.stream()
-                        .filter(diagnostic -> !jBallerinaBackend.diagnosticResult().diagnostics().contains(diagnostic))
-                        .collect(Collectors.toList());
-                if (!diagnostics.isEmpty()) {
-                    diagnostics.forEach(d -> out.println("\n" + d.toString()));
-                }
+            List<Diagnostic> emitDiagnostics = new ArrayList<>(emitResult.diagnostics().diagnostics());
```

Comment:
Why do we need to create a new list here? Can't we just use the returned collection and iterate over it and print the values?